### PR TITLE
More report stuff

### PR DESCRIPTION
**Pull request by jjkester**

### CODE REVIEW

File: Report/Chapters/Design.tex [hunk 1]
```diff
@@ -23,16 +23,44 @@ \subsubsection{Blocks and lines}
 This input block can be connected to other blocks to use the value as an input.
 Besides the most basic input block, the value block which takes a string as input and automatically detects the type, there is also a slider available.
 
+%\begin{figure}[h]
+%	\centering
+%	\includegraphics[scale=0.5]{Images/block-inputs}
+%	\label{fig:blocks-inputs}
+%	\caption{Different input blocks}
+%\end{figure}
```
Comment (wandernauta) [inline Report/Chapters/Design.tex]: This seems to be commented out, maybe it can be deleted?

File: Report/Chapters/Design.tex [hunk 2]
```diff
@@ -71,6 +126,29 @@ \subsubsection{Color scheme}
 
 \subsection{Behaviour and interaction design}
 
+Interaction with the user interface is possible through touch gestures on the touch screen and on mouse actions on the computer.
+We wanted the application to be very easy to use.
+Since JavaFX only supports three different touch events, we decided to only use the matching gestures drag, tap and tap-hold (or right click on a computer).
+With these three gestures everything is accessible in the program.
+More complicated gestures were considered, but implementation proved to be difficult, especially with multiple users in mind.
```
Comment (wandernauta) [inline Report/Chapters/Design.tex]: Could perhaps use an example.

**Comment by wandernauta** [comment]
Looks great! Will merge with the commented-out bit deleted, example is optional.

**Comment by wandernauta** [comment]
Really like the images, will merge.

**Comment by jjkester** [comment]
I think the text is fine without an example, it speaks for itself.

**Comment by wandernauta** [comment]
Merged, thanks a lot!

**Pull request merged**